platform-coop-toolkit/coop-library-framework
# Add Polylang meta synchronization and UI localization

## Description

This PR enables the following localization-specific features:

- Only desired metadata is synchronized between localized copies of a resource (fixes #14).
- The complete UI can now be localized using Polylang (see: https://polylang.pro/doc/developpers-how-to/#translate)

## Steps to test

1. Create a resource and populate the metadata fields.
2. Create a translation of the resource.

**Expected behavior:** The permanent link and translator fields should not be copied or synchronized.

## Additional information

This plugin also adds PHPUnit testing.

For plugin UI localization information, see:

- https://wordpress.org/plugins/theme-translation-for-polylang/

## Related issues

- Resolves #14
